db_sql.go

```diff
@@ -469,89 +539,155 @@ func SqlCreateDatabase(conx *DBConnection) error {
 	}
 }
 
-func MigrateSchema(conx *DBConnection, schema_name string, migrations []string) error {
-	return migrateSchemaInternal(true, conx, schema_name, migrations)
-}
-
-func migrateSchemaInternal(tryCreatingDB bool, conx *DBConnection, schema_name string, migrations []string) (migrateError error) {
-	if db, eConnect := conx.Connect(); eConnect != nil {
-		return eConnect
-	} else {
-		defer db.Close()
+func runMigrations(db *sql.DB, conx *DBConnection) (bool, error) {
+	// We have started using a migration tool by BurntSushi. This requires us to delete old version tracking tables etc.
```

Comment:
Avoid writing comments that pretend as though the only time is the present. When somebody reads "We have started using..." he has no idea when that happened. Rather phrase it as though you're writing history, for example:
"Authaus initially used it's own migration system, but in January 2016, we moved away from that, and instead started using the Burntsushi migration system. In order to make that change, we needed to...."